contracts/minter-suite/MinterHolderV0.sol

```diff
@@ -0,0 +1,494 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+import "../interfaces/0.8.x/IGenArt721CoreContractV1.sol";
+import "../interfaces/0.8.x/IMinterFilterV0.sol";
+import "../interfaces/0.8.x/IFilteredMinterHolderV0.sol";
+
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import "@openzeppelin/contracts/utils/structs/EnumerableMap.sol";
+
+pragma solidity 0.8.9;
+
+/**
+ * @title Filtered Minter contract that allows tokens to be minted with ETH
+ * when purchaser owns an allowlisted ERC-721 NFT.
+ * @author Art Blocks Inc.
+ */
+contract MinterHolderV0 is ReentrancyGuard, IFilteredMinterHolderV0 {
+    /**
+     * @notice Allowlisted holders of NFTs at address `_nftAddress` to be
+     * considered for minting.
+     */
+    event AllowlistedNftAddress(address indexed _nftAddress);
+
+    /**
+     * @notice Removed holders of NFTs at address `_nftAddress` to be
+     * considered for minting.
+     */
+    event RemovedNftAddress(address indexed _nftAddress);
```

Comment:
Nit: why not capitalize all of `NFT` in `{Allowlisted,Removed}NftAddress`?

Reply:
@jakerockland I can do that - should I do it everywhere? (e.g. `function removeNftAddress` -> `function removeNFTAddress`) or are you thinking only on these UpperCamelCase events?

Reply:
I think we should do it everywhere personally (using convention of always fully capitalizing acronyms). 🤷‍♂️

Reply:
perfect, updated!